crates/viewer/re_viewport_blueprint/src/viewport_blueprint.rs

```diff
@@ -572,12 +572,12 @@ impl ViewportBlueprint {
     /// Move the `contents` container or view to the specified target container and position.
     pub fn move_contents(
         &self,
-        contents: Contents,
+        contents: &[Contents],
```

Comment:
Better to pass by value if you're going to turn it into a Vec anyways
```suggestion
        contents: Vec<Contents>,
```